src/blog/v8-release-86.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'V8 release v8.6'
+author: 'Ingvar Stepanyan, a keyboard fuzzer'
```

Comment:
feel free to link to your Twitter here, like in other posts (unless you prefer not to)

Reply:
Ah right. Been a while :P

Reply:
```suggestion
author: '[Ingvar Stepanyan](https://twitter.com/RReverser), a keyboard fuzzer'
```